code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -197,42 +198,15 @@
 /datum/gear/uniform/pants/New()
 	..()
 	var/pants = list()
-	pants["jeans"] = /obj/item/clothing/under/pants
```

Comment:
put back jean shorts

Reply:
If the loss of jean shorts is unacceptable, I trust the devs to honor my request and close the PR, because I am not editing it.

Reply:
jorts are a sin but you should include their removal in the changelog, i think